aircmd/models/settings.py

```diff
@@ -21,17 +21,17 @@ def get_current_branch() -> str:
                                                                                                                                                                                                 
 def get_latest_commit_message() -> str:   
     repo = Repository(".")                                                                                                                                                                                                                                                                                                                      
-    commit: Commit = repo[repo.head.target]                                                                                                                                                              
+    commit: Commit = repo[os.environ.get("LAST_COMMIT_SHA", repo.head.target)]                                                                                                                                                             
     return str(commit.message)                                                                                                                                                                   
                                                                                                                                                                                                 
 def get_latest_commit_author() -> str:  
     repo: Repository = Repository(".")                                                                                                                                                                                                                                                                                                                       
-    commit: Commit = repo[repo.head.target]                                                                                                                                                              
+    commit: Commit = repo[os.environ.get("LAST_COMMIT_SHA", repo.head.target)]                                                                                                                                                              
```

Comment:
These are because Github likes to create its own merge commit so we can't simply take HEAD anymore, we actually have to either inject the sha from a github env variable or we have to take what's in HEAD if that is missing (e.g. local run)